#161342796 setup sass

ucheg6 opened the pull request:
#### What does this PR do?
setup sass preprocessing on the client
#### Description of Task to be completed?
- Install and setup sass pre-processing on the application to enables the use of both .scss and .css files on the client

#### How should this be manually tested?
- Git clone the repository
- cd into the `haven-ah-frontend` folder
- run `npm run start:dev`

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#161342796](https://www.pivotaltracker.com/story/show/161342796)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

ucheg6 committed:
ch(react-routers): Setup react router version 4 (React-router-dom)

install and configure react router creating a dummy route for homepage
create a component to test the route configuration

PriscillaSam commented:
This looks good @ucheg6. 👍

wiztemple reviewed (commented):
This is cool. well done @ucheg6

ucheg6 committed:
ch(sass/scss): Setup SASS processing on the client

Install and setup sass pre-processing to enable the use of both .scss and .css files on the client
